src/RA_Integration.vcxproj

```diff
@@ -158,7 +168,7 @@
     <ProjectName>RA_Integration</ProjectName>
     <ProjectGuid>{825CB292-F069-4B27-82CF-3417746ACDF3}</ProjectGuid>
     <RootNamespace>RA_Integration</RootNamespace>
-    <WindowsTargetPlatformVersion>7.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
```

Comment:
Why did this change?

Reply:
That is really weird, it says 7.0 on my computer.

Reply:
Yeah no problem on the document, I'll probably do another configuration later.